macro/run_simScript.py

```diff
@@ -123,6 +123,7 @@
 )
 
 parser.add_argument("--SND", dest="SND", help="Activate SND.", action='store_true')
+parser.add_argument("--SNDDesign", dest="SNDDesign", help="Choose SND design among [1,2,3,...]. 1: old version, 2: MTC", type=int, choices=[1, 2], required=False, default=1)
```

Comment:
```suggestion
parser.add_argument("--SNDDesign", help="Choose SND design among [1,2,3,...]. 1: old version, 2: MTC", type=int, choices=[1, 2], default=1)
```

I prefer not specifying keyword arguments which are left at their default values.